pkg/config/config.go

```diff
@@ -80,6 +80,34 @@ type Server struct {
 	CacheDictionarySize int               `deprecated:"true"`
 	CacheSegmentSize    int               `deprecated:"true"`
 	CacheTreeSize       int               `deprecated:"true"`
+
+	GoogleEnabled      bool   `def:"false" desc:"enables Google Oauth"`
+	GoogleClientID     string `def:"<yourClientID>" desc:"client ID generated for Google API"`
+	GoogleClientSecret string `def:"<yourClientSecret>" desc:"client secret generated for Google API"`
+	GoogleRedirectURL  string `def:"<pyroscope.mycompany.org/google/callback>" desc:"url that google will redirect to after logging in. Has to be in form <pathToPyroscopeServer/google/callback>"`
+	GoogleScopes       string `def:"https://www.googleapis.com/auth/userinfo.email" desc:"scopes for Google API"`
```

Comment:
At this point probably no, I left it in just in case someone could think of a use-case, would be easy to remove if you don't think there's anything useful we could offer?

Reply:
Yeah, I'd say let's remove them. Less flags is always better.